playground/platformsConfig.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { ReactNode } from "react";
+import {
+    DerivProductDerivTraderBrandLightLogoWordmarkHorizontalIcon,
+    DerivProductDerivBotBrandLightLogoWordmarkHorizontalIcon,
+    PartnersProductSmarttraderBrandLightLogoWordmarkIcon,
+    PartnersProductBinaryBotBrandLightLogoWordmarkHorizontalIcon,
```

Comment:
```suggestion
    DerivProductDerivTraderBrandLightLogoWordmarkHorizontalIcon as DerivTraderLogo,
    DerivProductDerivBotBrandLightLogoWordmarkHorizontalIcon as DerivBotLogo,
    PartnersProductSmarttraderBrandLightLogoWordmarkIcon as SmarttraderLogo,
    PartnersProductBinaryBotBrandLightLogoWordmarkHorizontalIcon as BinaryBotLogo,
```

just a suggestion, we can rename the long imports to small and more readable names to be used in JSX

Reply:
Thanks @shahzaib-deriv for your comment, this is just a test file and we won't include this in our build

Reply:
Sure @niloofar-deriv, I still believer we can impliment better solutions as it'll still be in the codebase.

Reply:
Base on this I'll update them 🙏